cmlreaders/readers/eeg.py

```diff
@@ -388,12 +388,14 @@ def as_timeseries(self,
                 if not reader.rereferencing_possible:
                     raise RereferencingNotPossibleError
                 data = self.rereference(data, contacts, scheme)
+                pairs = scheme[['contact_1', 'contact_2']].to_records().tolist()
 
             # TODO: tstart
             ts.append(
-                TimeSeries(data, sample_rate, epochs=epochs, contacts=contacts)
-            )
-            return TimeSeries.concatenate(ts)
+                TimeSeries(data, sample_rate, epochs=epochs,
+                           contacts=contacts if scheme is None else pairs)
```

Comment:
The intent of contacts was to actually be contacts. I don't think it makes sense to set it to pairs.

Reply:
Sure, but leaving it as contacts simply didn't work alongside rereferencing